test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/AzureFunctions/AzureFunctionTestFiles/AzureFunctionsInputBinding.ts

```diff
@@ -0,0 +1,59 @@
+using System;
```

Comment:
all the files in the AzureFunctionTestFiles folder are for initializing and verifying the tests